tests/presenters/breadcrumbs-presenter-test.php

```diff
@@ -0,0 +1,651 @@
+<?php
+/**
+ * WPSEO plugin test file.
+ */
+
+namespace Yoast\WP\SEO\Tests\Presenters;
+
+use Brain\Monkey;
+use Mockery;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Presentations\Indexable_Presentation;
+use Yoast\WP\SEO\Presenters\Breadcrumbs_Presenter;
+use Yoast\WP\SEO\Tests\TestCase;
+
+/**
+ * Class Breadcrumbs_Presenter_Test
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Presenters\Breadcrumbs_Presenter
+ *
+ * @group presenters
+ * @group breadcrumbs
+ */
+class Breadcrumbs_Presenter_Test extends TestCase {
+
+	/**
+	 * The options helper mock.
+	 *
+	 * @var Mockery\MockInterface|Options_Helper
+	 */
+	private $options;
+
+	/**
+	 * The breadcrumbs presenter instance.
+	 *
+	 * @var Breadcrumbs_Presenter
+	 */
+	private $instance;
+
+	/**
+	 * The indexable presentation.
+	 *
+	 * @var Indexable_Presentation
+	 */
+	private $presentation;
+
+	/**
+	 * Sets up the test class.
+	 */
+	public function setUp() {
+		$this->options = Mockery::mock( Options_Helper::class );
+
+		$this->instance     = Mockery::mock( Breadcrumbs_Presenter::class, [ $this->options ] )
+		                             ->makePartial()
+		                             ->shouldAllowMockingProtectedMethods();
```

Comment:
The tabs and spaces should not be mixed. This happens in a lot in this file (and perhaps other tests too).
You can change the PHPStorm settings: Editor --> Code Style --> PHP --> Wrapping and Braces -> Chained methods calls to `Chop down if long` and uncheck `Align when multiline`. Then the auto-format will do this for you correctly.

Reply:
Fixed in own commit